docs/src/api/commandline.md

```diff
@@ -0,0 +1,221 @@
+`uniffi-bindgen-react-native` the command line utility that ties together much of the building of Rust, and the generating the bindings and turbo-modules.
+
+Most commands take a `--config FILE` option. This is a YAML file which collects commonly used options together, and is [documented here](config-yaml.md).
+
+The command can be invoked with `./node_modules/.bin/uniffi-bindgen-react-native`.
+
+Running the following command from within your project:
+
+```sh
+alias ubrn=$(pwd)/node_modules/.bin/uniffi-bindgen-react-native
```

Comment:
Might be useful to include a comment about mono-repo support, or suggest they use this command instead which should work in both repo structures

`UBRN_BIN=$(node -p "require('path').resolve(require.resolve('uniffi-bindgen-react-native/package.json'), '../../.bin/uniffi-bindgen-react-native')")`